docs/content/preview/yugabyte-platform/manage-deployments/xcluster-replication/_index.md

```diff
@@ -0,0 +1,142 @@
+
+{{<lead link="https://www.yugabyte.com/blog/distributed-database-transactional-consistency-async-standby">}}
+Blog: [Can Distributed Databases Achieve Transactional Consistency on Async Standbys? Yes, They Can](https://www.yugabyte.com/blog/distributed-database-transactional-consistency-async-standby/)
+{{</lead>}}
+
+{{<lead link="https://www.youtube.com/watch?v=lI6gw7ncBs8/">}}
+Video: [YFTT - Transactional xCluster](https://www.youtube.com/watch?lI6gw7ncBs8)
+{{</lead>}}
+
+{{<index/block>}}
+
+  {{<index/item
+    title="Set up xCluster replication"
+    body="Designate a universe to act as a source."
+    href="xcluster-replication-setup/"
+    icon="fa-light fa-copy">}}
+
+  {{<index/item
+    title="Manage tables and indexes"
+    body="Perform DDL changes to databases in replication."
+    href="xcluster-replication-ddl/"
+    icon="fa-light fa-table">}}
+
+  {{<index/item
+    title="Bidirectional"
+    body="Configure bidirectional replication."
+    href="bidirectional-replication/"
+    icon="fa-light fa-arrows-left-right">}}
+
+{{</index/block>}}
+
+## Limitations
+
+- Currently, replication of DDL (SQL-level changes such as creating or dropping tables or indexes) is not supported. To make these changes requires first performing the DDL operation (for example, creating a table), and then adding the new object to replication in YugabyteDB Anywhere. Refer to [Manage tables and indexes](./xcluster-replication-ddl/).
+
+- xCluster replication setup (and other operations that require making a full copy from source to target, such as adding tables with data to replication, resuming replication after an extended network outage, and so on) forcefully drop the tables on the target if they exist before performing the restore.
+
+    If there are any open SQL connections to the database on the target, they will be interrupted and you should retry the connection.
+
+- Setting up xCluster replication between a universe earlier than or upgraded to v2.20.x and a new v2.20.x universe is not supported. This is due to a limitation of xCluster deployments and packed rows. See [Packed row limitations](../../../architecture/docdb/packed-rows/#limitations).
+
+- You can set up change data capture on a source universe in xCluster replication, but not a target.
+
+- xCluster replication is not supported for materialized views.
+
+## Upgrading universes in xCluster replication
+
+Use the same version of YugabyteDB on both the source and target.
+
+When [upgrading universes](../../manage-deployments/upgrade-software-install/) in xCluster replication, you should upgrade and finalize the target before upgrading and finalizing the source.
+
+If you upgrade and finalize the source universe first, replication may be paused automatically until both universes are finalized to the same software version.
+
+## xCluster DR vs xCluster Replication
```

Comment:
As we discussed, this can live on the DR page and link to the main content there with a few sentences here.